fv3gfs/util/communicator.py

```diff
@@ -451,6 +452,10 @@ def halo_update(self, quantity: Quantity, n_points: int):
         req = self.start_halo_update(quantity, n_points)
         req.wait()
 
+    def _synchronize(self):
+        """Wait for all work that could be in-flight to finish."""
+        device_synchronize(None)
```

Comment:
Since this doesn't use any information from `self` and is one line, can you instead call `device_synchronize` directly where this is called? Or do you plan to extend this routine in the future?

If you keep this method, `Communicator._synchronize` carries certain connotations, since communicators do synchronize data. Can you extend the name to `_device_synchronize` so that it's clearer this is handling GPU synchronization?

Reply:
So I need a function so that I can track the call from outside using the python profiler hook.
I renamed it and made it a `@staticmethod`